dataprep/eda/dtypes.py

```diff
@@ -233,3 +255,26 @@ def is_pandas_categorical(dtype: Any) -> bool:
     Detect if a dtype is categorical and from pandas.
     """
     return any(isinstance(dtype, c) for c in CATEGORICAL_PANDAS_DTYPES)
+
+
+def drop_null(
+    var: Union[pd.Series, dd.Series, pd.DataFrame, dd.DataFrame]
+) -> Union[pd.Series, dd.Series, pd.DataFrame, dd.DataFrame]:
+    """
+    Parameters : Pandas or Dask DataFrame/Series
+    returns a Pandas/Dask Dataframe or Serieswith N/A or Null values removed.
+    """
+    # creating a union of series and dataframe types
+    x = Union[pd.Series, dd.Series]
+    y = Union[pd.DataFrame, dd.DataFrame]
+
+    if isinstance(var, x.__args__):
+        return var[~var.isin(NULL_VALUES)]
+
+    elif isinstance(var, y.__args__):
+        df = var
+        for values in var.columns:
+            df = df[~df[values].isin(NULL_VALUES)]
+        return df
+
+    return None
```

Comment:
Can you please change this function to

``` python

def drop_null(
    var: Union[pd.Series, dd.Series, pd.DataFrame, dd.DataFrame]
) -> Union[pd.Series, dd.Series, pd.DataFrame, dd.DataFrame]:
    """
    Drop the null values (specified in NULL_VALUES) from a series or DataFrame
    """

    if isinstance(var, (pd.Series, dd.Series)):
        if is_datetime(var.dtype):
            return var.dropna()
        return var[~var.isin(NULL_VALUES)]

    elif isinstance(var, (pd.DataFrame, dd.DataFrame)):
        df = var
        for values in df.columns:
            if is_datetime(df[values].dtype):
                df = df.dropna(subset=[values])
            else:
                df = df[~df[values].isin(NULL_VALUES)]
        return df

    return None
```

`isin` does not work for datetime columns. However, if a column is datetime with errors, they will necessarily be marked as NaT, therefore, `dropna()` will remove all values that don't match the specified datetime format in the column.

Reply:
Thanks for the pointing this out! I have updated the function.